core/templates/dev/head/css/oppia.css

```diff
@@ -303,6 +303,10 @@ body {
   z-index: 10;
 }
 
+.oppia-learner-dashboard-side-content{
```

Comment:
Hi @rohitkatlaa,  PTAL the lint test failing.
Add a space before '{' Thanks!

Reply:
Also, you can move the `min-width` style to this class.

Reply:
I have moved min-width.